src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -374,6 +377,7 @@ export class InstructorCourseEditPageComponent implements OnInit {
     });
 
     panelDetail.editPanel.isEditing = false;
+    panelDetail.originalPanel = panelDetail.editPanel;
```

Comment:
Why saving instructor need to restore the panel?

Reply:
It is to update the copy with the new saved values